cdap-docs/build.sh

```diff
@@ -30,6 +30,34 @@ ARG_1="${1}"
 ARG_2="${2}"
 ARG_3="${3}"
```

Comment:
Positional parameters are positional parameters. There's no need to wrap them in quotes, even if they includes spaces and are quoted in the input.